packages/hardhat-core/src/internal/hardhat-network/provider/output.ts

```diff
@@ -202,6 +234,9 @@ export function getRpcReceipts(
           : null,
       logs,
       logsBloom: bufferToRpcData(receipt.bitvector),
+      type: showTransactionType
```

Comment:
Why there is no `|| tx instanceof AccessListEIP2930Transaction` here?

Reply:
Added a comment